dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
+    return [] if level_source.nil?
+
+    responses = JSON.parse(level_source.data)
+    learning_module_weights = Hash.new(0)
+
+    responses.each do |level_id, response|
+      level = EvaluationMulti.find(level_id)
+      selected_answer = level.answers[response['result'].to_i]
```

Comment:
The big reason for `LevelGroup` to own this is that we can change the implementation of how the answers are stored (see https://github.com/code-dot-org/code-dot-org/pull/8661/files#r65259283) without affecting the consumers.